tick_scripts/print_tick_scripts.py

```diff
@@ -0,0 +1,92 @@
+from lsst.ts.xml.component_info import ComponentInfo
+
+tick_script_template = """HVAC-{name}-Alarm
+
+=====================================
+var db = 'efd'
+var rp = 'autogen'
+var measurement = 'lsst.sal.HVAC.{topic}'
+var groupBy = []
+var whereFilter = lambda: {where}
+var name = 'HVAC {name} Alarm'
+var idVar = name
+var message = '{{ if eq .Level "OK" }}Summit EFD HVAC {name} is back to normal.{{ else }}Summit EFD HVAC
+{name} alarm. <@U07PRCW6JKT>, <@U01EF071X7U>, <@U07SCNZE90S>, <@U07N05MTGD8>, <@U07PMHC954K> please check.
+{{ end }}'
+var idTag = 'alertID'
+var levelTag = 'level'
+var messageField = 'message'
+var durationField = 'duration'
+var outputDB = 'chronograf'
+var outputRP = 'autogen'
+var outputMeasurement = 'alerts'
+var triggerType = 'threshold'
+
+var data = stream
+    |from()
+        .database(db)
+        .retentionPolicy(rp)
+        .measurement(measurement)
+        .groupBy(groupBy)
+        .where(whereFilter)
+
+var trigger = data
+    |alert()
+        .crit(lambda: {crit})
+        .stateChangesOnly()
+        .message(message)
+        .id(idVar)
+        .idTag(idTag)
+        .levelTag(levelTag)
+        .messageField(messageField)
+        .durationField(durationField)
+        .stateChangesOnly()
+        .post('<PUT SQUADCAST URL HERE>')
```

Comment:
Should this be changed for the squadcast url?

Reply:
Before you run the script you need to replace that with the real URL, which needs to remain private.